test/chunkserver/chunkserver_snapshot_test.cpp

```diff
@@ -557,14 +557,7 @@ TEST_F(ChunkServerSnapshotTest, TwoNodesShutdownOnePeer) {
         shutdownPeerid = peer1;
     }
     ASSERT_EQ(0, cluster.ShutdownPeer(shutdownPeerid));
-    // 测试发现集群不可用
-    ReadVerifyNotAvailable(leaderId,
```

Comment:
should we test read can be executed in this case, and sleep a while, then read cann't be performed because the leader is step down.

Reply:
how can I get a sleep time? we need sleep for a election time, get it from `copyset.election_timeout_ms=1000` from config or following way:

https://github.com/opencurve/curve/blob/bdc8f89a6563a2560fbc079bfc5b5908e148e8ee/test/chunkserver/client.cpp#L37

https://github.com/opencurve/curve/blob/bdc8f89a6563a2560fbc079bfc5b5908e148e8ee/test/chunkserver/client.cpp#L117-L118